pkg/kubelet/cadvisor.go

```diff
@@ -59,7 +59,7 @@ func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, container
 	if cc == nil {
 		return nil, nil
 	}
-	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.dockerClient, false)
+	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.containerRuntime, false)
```

Comment:
Does every kubelet in a k8s cluster use the same runtime?

Sent from my iPhone

> On Feb 13, 2015, at 5:58 PM, Yifan Gu <notifications@github.com> wrote:
> 
> In pkg/kubelet/cadvisor.go:
> 
> > @@ -59,7 +59,7 @@ func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, container
> >  	if cc == nil {
> >  		return nil, nil
> >  	}
> > -	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.dockerClient, false)
> > +	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.containerRuntime, false)
> @thockin IMO, since docker and rocket are going to support both docker image and aci. I think one container runtime is enough, and I think we can spec which one to run in the configure :)
> 
> —
> Reply to this email directly or view it on GitHub.
>